src/dosbox.cpp

```diff
@@ -446,15 +446,22 @@ void DOSBOX_Init(void) {
 	Pint->SetMinMax(0,10);
 	Pint->Set_help("How many frames DOSBox skips before drawing one.");
 
-	Pbool = secprop->Add_bool("aspect",Property::Changeable::Always,false);
-	Pbool->Set_help("Do aspect correction, if your output method doesn't support scaling this can slow things down!");
+	Pbool = secprop->Add_bool("aspect", Property::Changeable::Always, true);
+	Pbool->Set_help("Do aspect correction. Affects display modes using non-square\n"
+	                "pixels (such as 320x200 or 640x400), scaling them to maintain\n"
+	                "4:3 aspect ratio the most games were designed for.\n"
+	                "Change to false to use square pixels in all modes.");
```

Comment:
"_Do aspect correction_" sounds a bit awkward, given there's a verb a the front and back of the sentence. We can switch to the active-voice by dropping _do_, and starting with either _correct_, or more specifically _scale_.  I've reordered the sentences:

"Scales the vertical resolution to produce a 4:3 display aspect ratio, matching that of original standard definition monitors for which DOS games were designed. This setting only affects video modes that use non-square pixels, such as 320x200 or 640x400; where as modes such as 640x480 and 800x600 will be displayed as-is. "

Reply:
Used exactly this description - feels way more natural than the sentence I copied/modified from dosbox wiki.